Submissions/Ekey_Week2 Simple Values and Control Flow.playground/Contents.swift

```diff
@@ -0,0 +1,72 @@
+//: Simple Values
+//: 1. Create a constant with an explicit type of Float that equals 4.
+let explicitFloat: Float = 4.0
+
+//: 2. Try removing the conversion to String from the last line. Copy the error and place it in a block comment below this block of code.
+let label = "The width is "
+let width = 94
+let widthLabel = label + String(width)
+
+//  let widthLabel = label + (width)
+//  Binary operator '+' cannot be applied to operands of type 'String' and 'Int'
+
+//: 3. Use \() to include a floating-point calculation in a string and to include someone's name in a greeting.
+let myName = "Laurie"
+let Greeting = "Hello my name is "
+let age: Float = 52.1
+let assignment1 = "\(Greeting)\(myName) and I am OLD, \(age)"
```

Comment:
You were instructed to include `calculation` in your string interpolation, such as `\(age + futureDate) `